distribution/delete_test.go

```diff
@@ -122,13 +124,26 @@ func TestDeleteModel(t *testing.T) {
 			}
 
 			// Attempt to delete the model and check for expected error
-			if err := client.DeleteModel(tc.ref, tc.force); !errors.Is(err, tc.expectedErr) {
+			out, err := client.DeleteModel(tc.ref, tc.force)
+			if !errors.Is(err, tc.expectedErr) {
 				t.Fatalf("Expected error %v, got: %v", tc.expectedErr, err)
 			}
 			if tc.expectedErr != nil {
 				return
 			}
 
+			if slices.Contains(tc.tags, tc.ref) {
+				// tc.ref is a tag
+				require.Equal(t, "Untagged: "+tc.ref+"\n", out)
+			} else {
+				// tc.ref is an ID
+				expectedOut := ""
+				for _, tag := range tc.tags {
+					expectedOut += "Untagged: " + tag + "\n"
+				}
+				require.Equal(t, expectedOut, out)
```

Comment:
Could you use a string comparison and fail if not equals? 
We were trying to keep the deps as minimum and we agreed with @ekcasey to try to not use any lib for testing 🙏